spec/lib/diaspora_federation/entities/account_rename_spec.rb

```diff
@@ -0,0 +1,38 @@
+module DiasporaFederation
+  describe Entities::AccountRename do
+    let(:data) { FactoryGirl.attributes_for(:account_rename_entity) }
+
+    let(:xml) { <<-XML }
+<account_rename>
+  <author>#{data[:author]}</author>
+  <person>
+    <guid>#{data[:person].guid}</guid>
+    <diaspora_handle>#{data[:person].author}</diaspora_handle>
+    <url>#{data[:person].url}</url>
+    <profile>
+      <diaspora_handle>#{data[:person].profile.author}</diaspora_handle>
+      <first_name>#{data[:person].profile.first_name}</first_name>
+      <last_name/>
+      <image_url>#{data[:person].profile.image_url}</image_url>
+      <image_url_medium>#{data[:person].profile.image_url}</image_url_medium>
+      <image_url_small>#{data[:person].profile.image_url}</image_url_small>
+      <birthday>#{data[:person].profile.birthday}</birthday>
+      <gender>#{data[:person].profile.gender}</gender>
+      <bio>#{data[:person].profile.bio}</bio>
+      <location>#{data[:person].profile.location}</location>
+      <searchable>#{data[:person].profile.searchable}</searchable>
+      <nsfw>#{data[:person].profile.nsfw}</nsfw>
+      <tag_string>#{data[:person].profile.tag_string}</tag_string>
+    </profile>
+    <exported_key>#{data[:person].exported_key}</exported_key>
+  </person>
+</account_rename>
+    XML
```

Comment:
Nit: Remove spaces before `XML` (yeah, I know, IDE ... I really need to figure out how to prevent rubymine from doing this ;) )